tests/anlogic/run-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+{
+echo "all::"
+for x in *.ys; do
+	echo "all:: run-$x"
+	echo "run-$x:"
+	echo "	@echo 'Running $x..'"
+	echo "	@../../yosys -ql ${x%.ys}.log $x -w 'Yosys has only limited support for tri-state logic at the moment.'"
```

Comment:
Please move $x at end of line, like in https://github.com/YosysHQ/yosys/pull/1409/commits/7f0eec8270dbef06e3e6970af20dae2d6f89f6b9

Reply:
done